test/extended/baremetal/hosts.go

```diff
@@ -186,4 +186,64 @@ var _ = g.Describe("[sig-installer][Feature:baremetal][Serial] Baremetal platfor
 		o.Expect(err).NotTo(o.HaveOccurred())
 		o.Expect(found).To(o.BeFalse())
 	})
+
+	g.It("successfully provision resources when virtualMediaViaExternalNetwork is used", func() {
+		skipIfNotBaremetal(oc)
+
+		if !helper.CanDeployExtraWorkers() {
+			e2eskipper.Skipf("No extra worker is detected for provisioning")
+		}
+
+		dc := oc.AdminDynamicClient()
+		prvClient := provisioningClient(dc)
+
+		provisionings, err := prvClient.List(context.Background(), metav1.ListOptions{})
+		o.Expect(err).NotTo(o.HaveOccurred())
+		o.Expect(provisionings.Items).NotTo(o.BeEmpty())
+
+		// expect only one provision config
+		provision := provisionings.Items[0]
+
+		realVal, found, err := unstructured.NestedBool(provision.Object, "spec", "virtualMediaViaExternalNetwork")
```

Comment:
This test seems to impact quite deeply the environment, thus I guess it'd be better to move it in the serial suite to avoid affecting other tests?

Reply:
This test is serial suite test which is running under https://github.com/openshift/origin/blob/f8e71ecca3a89c95ecbf0a11f77f434b2fe70c63/test/extended/baremetal/hosts.go#L154